model-optimizer/mo/utils/schema.json

```diff
@@ -0,0 +1,111 @@
+{
+	"definitions": {},
+	"$schema": "http://json-schema.org/draft-07/schema#", 
+	"$id": "https://example.com/object1624983563.json", 
+	"title": "Root", 
+	"type": "array",
+	"default": [],
+	"items":{
+		"$id": "#root/items", 
+		"title": "Items", 
+		"type": "object",
+		"properties": {
+			"custom_attributes": {
+				"$id": "#root/items/custom_attributes", 
+				"title": "Custom_attributes", 
+				"type": "object",
+				"properties": {
+				}
+			}
+,
+			"id": {
+				"$id": "#root/items/id", 
+				"title": "Id", 
+				"type": "string",
+				"default": "",
+				"pattern": "^.*$"
+			},
+			"inputs": {
+				"$id": "#root/items/inputs", 
+				"title": "Inputs", 
+				"type": "array",
+				"default": [],
+				"items":{
+					"$id": "#root/items/inputs/items", 
+					"title": "Items", 
+					"type": "array",
+					"default": [],
+					"items":{
+						"$id": "#root/items/inputs/items/items", 
+						"title": "Items", 
+						"type": "object",
+						"properties": {
+							"node": {
+								"$id": "#root/items/inputs/items/items/node", 
+								"title": "Node", 
+								"type": "string",
+								"default": "",
+								"pattern": "^.*$"
```

Comment:
will ".*" mean the same? Since "." in json schema means everything except line break (platform specific). 

Can it be an empty string? The current rule allows it.

Reply:
Yes. Validator allows empty string for inputs.